tests/testAfterburners.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+
+import unittest
+import lsst.utils.tests as utilsTests
+
+import lsst.meas.base.afterburner as AB
```

Comment:
Don't think the code standards would smile on `AB`.  I've used something like `mbAfterburner` in the past, but dunno what is recommended.  How about just `afterburner`, or just pull out the symbols you need, or import the symbols into `lsst.meas.base` in the module's \_\_init\_\_.py?